### ISSUE TITLE
Incorrect info in this doc...

### ISSUE THREAD

**Issue by PJMarcum** [MicrosoftDocs/memdocs#1551]
This sentence, "Password expiration (days): Enter the number of days before the password expires, and they must create a new one, from 1-730." is wrong..... In the Intune UI it says 1-365, to test I entered a number higher than 365 and the UI will not accept the value. 

---
#### Document Details

⚠ *Do not edit this section. It is required for docs.microsoft.com ➟ GitHub issue linking.*

* ID: 979d9182-7aa8-9a97-399b-2a495164f1a7
* Version Independent ID: b293cc1c-53ce-d06f-c991-a0f1a27214fc
* Content: [Windows 10 compliance settings in Microsoft Intune - Azure](https://docs.microsoft.com/en-us/mem/intune/protect/compliance-policy-create-windows)
* Content Source: [memdocs/intune/protect/compliance-policy-create-windows.md](https://github.com/MicrosoftDocs/memdocs/blob/master/memdocs/intune/protect/compliance-policy-create-windows.md)
* Service: **microsoft-intune**
* Sub-service: **protect**
* Technology: ****
* GitHub Login: @Brenduns
* Microsoft Alias: **brenduns**

**Comment by Brenduns** [comment]
@PJMarcum   Thanks for this submission.   Unfortunately, its not clear what's going on for your tenant, but that isn't an issue I can reproduce locally. I've checked that setting on my own public tenant, and some internal test tenants just now. For each, the UI text references up to 730 and the UI supports the same, a configuration of up to 730 days. It does error when I use zero or 731 or greater.

I've pasted a screen capture of that below that I just took from my live tenant.  If your UI displays and supports something different, we'll need you to  [contact support](https://docs.microsoft.com/mem/get-support)  to have that investigated. That's not something we can assist on via this doc feedback channel. 

![image](https://user-images.githubusercontent.com/17034749/115788273-42612d80-a378-11eb-9f23-0e140673a546.png)
